common/app/renderers/DotcomRenderingService.scala

```diff
@@ -137,6 +135,27 @@ class DotcomRenderingService extends GuLogging with ResultWithPreconnectPreload
     post(ws, json, Configuration.rendering.baseURL + "/Article", page)
   }
 
+  def getBlocks(
+      ws: WSClient,
+      page: LiveBlogPage,
+      blocks: Seq[Block],
+  )(implicit request: RequestHeader): Future[String] = {
+
+    val dataModel = DotcomBlocksRenderingDataModel(page, request, blocks)
+    val json = DotcomBlocksRenderingDataModel.toJson(dataModel)
+
+    ws.url(Configuration.rendering.baseURL + "/Blocks")
```

Comment:
would it be possible to reuse the post method? I would keep this as a thin service layer that just calls DCR and gets back a Future[Result], then parse/deal with that where we use it (in getDCRBlocksHTML I think)

Reply:
It'll be difficult to use/work with a Future[Result], for the other methods in this class I think the general expectation is that we return the Result from frontend without touching it, however for Blocks its slightly different as we only want the HTML from DCR(without any of the headers, status code, etc) so that we can insert it into a JSON blob.

I've refactored it a bit by making a postWithoutHandler method which is used by both `post` and `getBlocks` so that theres a bit less duplication.